autoprotocol/constants.py

```diff
@@ -35,3 +35,5 @@
     (0.6243159222654036, -0.14191423819668741),
     (0.563207308766221, 0.5152506858187523),
 ]
+
+MEASUREMENT_MODES = ["mass", "volume"]
```

Comment:
nit: Let's rename this to `PROVISION_MEASUREMENT_MODES`, since there are other measurement modes (e.g. Spectrophotometry)

Reply:
Renamed.